components/nimbus/src/persistence.rs

```diff
@@ -442,6 +559,503 @@ mod tests {
         assert_ne!(fs::metadata(&db_file)?.len(), garbage_len);
         Ok(())
     }
+
+    // fn get_valid_feature_experiments() -> Vec<serde_json::Value> {
+    //     vec![json!({
+    //         "schemaVersion": "1.0.0",
+    //         "slug": "secure-gold", // change when cloning
+    //         "endDate": null,
+    //         "featureIds": ["abc"], // change when cloning
+    //         "branches":[
+    //             {
+    //                 "slug": "control",
+    //                 "ratio": 1,
+    //                 "feature": {
+    //                     "featureId": "abc", // change when cloning
+    //                     "enabled": false
+    //                 }
+    //             },
+    //             {
+    //                 "slug": "treatment",
+    //                 "ratio":1,
+    //                 "feature": {
+    //                     "featureId": "abc", // change when cloning
+    //                     "enabled": true
+    //                 }
+    //             }
+    //         ],
+    //         "channel": "nightly",
+    //         "probeSets":[],
+    //         "startDate":null,
+    //         "appName": "fenix",
+    //         "appId": "org.mozilla.fenix",
+    //         "bucketConfig":{
+    //             // Setup to enroll everyone by default.
+    //             "count":10_000,
+    //             "start":0,
+    //             "total":10_000,
+    //             "namespace":"secure-gold", // change when cloning
+    //             "randomizationUnit":"nimbus_id"
+    //         },
+    //         "userFacingName":"Diagnostic test experiment",
+    //         "referenceBranch":"control",
+    //         "isEnrollmentPaused":false,
+    //         "proposedEnrollment":7,
+    //         "userFacingDescription":"This is a test experiment for diagnostic purposes.",
+    //         "id":"secure-gold", // change when cloning
+    //         "last_modified":1_602_197_324_372i64
+    //     })]
+    // }
```

Comment:
If this isn't being used, let's get rid of it.

Reply:
All the stuff that's commented out will either be morphed into something that is used or removed before landing.